batocera-es-swissknife: use hotkeygen, declutter... - v43

crcerror opened the pull request.

nadenislamarre commented:
timeout 5 hotkeygen --send exit && return 20

i think that this command always returns immediatly, at least, it will not wait that the game do somethiing.
it just simulate a keyboard pressure, it doesn't call the kill or anything that is done asynchronously behing by evmapy

crcerror commented:
I squeezed the code, and yes @nadenislamarre you were right. `hotkeygen` does not give a return value.... So I monitor the runcommand.py.

Addition:
Cleaned up code base
Added timeout as user option
Updated help

crcerror commented:
Optimized version is right here.... it can uses fractions in s for timeout and is well tested now. So please ready for current v42
